tests/integration_tests_control_files/time_refinement_study/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+set(TEST_OUTPUT_DIR ${CMAKE_CURRENT_BINARY_DIR})
+
+# =======================================
+# Time Study (Linear Advection Explicit RK)
+# =======================================
+# ----------------------------------------
+# - details: [if needed]
+# ----------------------------------------
+configure_file(time_study_advection_explicit.prm time_study_advection_explicit.prm COPYONLY)
+add_test(
+    NAME 1D_TIME_STUDY_ADVECTION_EXPLICIT
```

Comment:
`1D_TEMPORAL_CONVERGENCE_STUDY_ADVECTION_EXPLICIT` instead?

Reply:
Renaming to `1D_TIME_REFINEMENT_STUDY_ADVECTION_EXPLICIT` - I've been using the naming "time refinement" and it makes sense to be consistent. Will also change the .prm file name to match.